### DISCUSSION TITLE
Add control of label sizing and placement

### DISCUSSION THREAD

**Discussion by amruelkomy** [grafana/grafana#67245] [Dashboards, Panels & Visualizations]
Please support options for controlling the labels size and placement in pie chart. maybe by dragging the label to the right placement. and the size can be changed from options.

**Comment by OmarAlkttan** [comment]
I have same problem, please fix this soon.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82743. Feel free to continue the discussion around this there. Thank you!
